src/status_im/ui/screens/profile/events.cljs

```diff
@@ -172,17 +171,26 @@
             (status-change {:old-status old-status
                             :status     status}))))))
 
+(handlers/register-handler-fx
+  :my-profile/start-editing-profile
+  (fn [{:keys [db]} []]
+    {:db (assoc db :my-profile/editing? true)}))
+
 (handlers/register-handler-fx
   :my-profile/save-profile
   (fn [{:keys [db now]} _]
     (let [{:accounts/keys [accounts current-account-id]} db
           {old-status :status} (get accounts current-account-id)
           {:keys [status photo-path]} (:my-profile/profile db)
           cleaned-name (clean-name db :my-profile/profile)
-          cleaned-edit {:name         cleaned-name
-                        :status       status
-                        :photo-path   photo-path
-                        :last-updated now}]
+          cleaned-edit (if photo-path
+                         {:name         cleaned-name
+                          :status       status
```

Comment:
Can we do this without the redundancy?

Reply:
(if photo-path (assoc cleaned-edit :photo-path photo-path) cleaned-edit)